tools/send-to-caas/bulk-publish-to-caas.beta.js

```diff
@@ -147,7 +147,7 @@ const showErrorTable = (errorArr) => {
     tableBody.innerHTML += `<tr>
       <td>${index}</td>
       <td class="error">Failed</td>
-      <td><a href="${pageUrl}">${pageUrl}</a></td>
+      <td><a href="${pageUrl}" target="_blank">${pageUrl}</a></td>
```

Comment:
This is a good usability improvement for the UI. Could add `rel=noopener noreferrer` so the new tab can't access `window.opener`